config-default.yml

```diff
@@ -322,6 +323,11 @@ anti_spam:
             max: 3
 
 
+anti_malware:
+    whitelist: ['.bmp', '.gif', '.jpg', '.jpeg', '.png', '.tiff', # Images
```

Comment:
I think the [more conventional syntax](https://yaml.org/spec/1.2/spec.html#id2759963) for lists should be used as it's more consistent with the rest of the config.